lib/plug/conn.ex

```diff
@@ -690,6 +690,17 @@ defmodule Plug.Conn do
   end
 
   @doc """
+  Clears the entire session.
+
+  This function removes every key from the session, effectively clearing the
```

Comment:
Can we add a note though that the session is still send to the client if the user wants to effectively drop the session they should use configure_session with :drop?